src/server.rs

```diff
@@ -1012,7 +988,66 @@ impl WebThingServer {
                                 web::resource("/events/{event_name}")
                                     .route(web::get().to(handle_get_event)),
                             ),
-                    )
+                    );
+            }
+        }
+    }
+
+    /// Start listening for incoming connections.
+    pub fn start(
+        &mut self,
+        configure: Option<Arc<dyn Fn(&mut web::ServiceConfig) + Send + Sync + 'static>>,
+    ) -> Server {
+        let port = self.port.unwrap_or(80);
+
+        let mut hosts = vec!["localhost".to_owned(), format!("localhost:{}", port)];
```

Comment:
The code that builds `hosts` can be removed entirely, `hosts` is used nowhere in the `start` function.